include/pros/imu.h

```diff
@@ -74,6 +74,27 @@ typedef struct __attribute__((__packed__)) euler_s {
  */
 int32_t imu_reset(uint8_t port);
 
+
+/**
+ * Set the Inertial Sensor's refresh interval in milliseconds.
+ *
+ * The rate may be specified in increments of 5ms, and will be rounded down to
+ * the nearest increment. The minimum allowable refresh rate is 5ms.
```

Comment:
```suggestion
 * the nearest increment. The minimum allowable refresh rate is 5ms. The default 
 * rate is 10ms. Setting to less than 10ms will not increase the refresh rate on the 
 * user side, which is 10ms. However, it will guarantee that the data is more recent.
```
I guessed the wrap.